test/integration/test-visibility-states.js

```diff
@@ -114,7 +114,7 @@ t.run('Viewer Visibility State', () => {
             docHidden = sandbox.stub(docState, 'isHidden').returns(false);
 
             resources = Services.resourcesForDoc(win.document);
-            doPass_ = resources.doPass;
+            doPass_ = resources.doPass_;
```

Comment:
This will fail, because the integration test will mangle the property name. That's why we made it public to begin with. Maybe rename the method to make it clear isn't not to be used by anyone else?

Reply:
reverted changes. added visibleForTesting annotation instead. I think we should clean up the test. It's weird to have a such hybrid of unit test & integration test.